src/nft/components/profile/list/NFTListingsGrid.tsx

```diff
@@ -0,0 +1,550 @@
+            <Box as="img" width="32" src="/nft/svgs/minusCircle.svg" alt="Remove item" />
+          </Box>
+          <Box
+            as="img"
+            alt={selectedMarkets[0].name}
+            width="28"
+            height="28"
+            borderRadius="4"
+            objectFit="cover"
+            src={selectedMarkets[0].icon}
+            marginRight="16"
+          />
+        </Column>
+      )}
+      <Column style={{ flex: '1.5' }}>
+        {globalPriceMethod === SetPriceMethod.SAME_PRICE && !globalOverride ? (
+          <PriceTextInput
+            listPrice={globalPrice}
+            setListPrice={setGlobalPrice}
+            isGlobalPrice={true}
+            setGlobalOverride={setGlobalOverride}
+            globalOverride={globalOverride}
+            warning={warning}
+            asset={asset}
+          />
+        ) : (
+          <PriceTextInput
+            listPrice={listPrice}
+            setListPrice={setListPrice}
+            isGlobalPrice={false}
+            setGlobalOverride={setGlobalOverride}
+            globalOverride={globalOverride}
+            warning={warning}
+            asset={asset}
+          />
+        )}
+      </Column>
+      <Row flex="1" display={{ sm: 'none', md: 'flex' }}>
+        <Box className={body} color="textSecondary" width="full" textAlign="right">
+          {marketplaceFee > 0 ? marketplaceFee + (selectedMarkets.length > 1 ? '% MAX' : '%') : '--%'}
+        </Box>
+      </Row>
+      <Row flex="1" display={{ sm: 'none', md: 'flex' }}>
+        <Box className={body} color="textSecondary" width="full" textAlign="right">
+          {(asset.basisPoints * 0.01).toFixed(1)}%
+        </Box>
+      </Row>
+      <Row style={{ flex: '1.5' }} display={{ sm: 'none', md: 'flex' }}>
+        <Column width="full">
+          <EthPriceDisplay ethPrice={userReceives} />
+          {(showGlobalPrice ? globalPrice && globalPrice !== 0 : listPrice !== 0) && (
+            <Row marginTop="4" width="full" fontSize="12" color="textSecondary">
+              {profit ? <Box marginLeft="auto">Profit: {formatEth(profit)} ETH</Box> : null}
+              {profitPercent ? (
+                <Box marginLeft="8" marginRight="0">
+                  ({profitPercent > 0 && '+'}
+                  {profitPercent > 1000 ? Math.round(profitPercent / 1000) + 'K' : profitPercent}%)
+                </Box>
+              ) : null}
+            </Row>
+          )}
+        </Column>
+      </Row>
+    </Row>
+  )
+}
+
+export interface NFTListRowProps {
+  asset: WalletAsset
+  globalPriceMethod?: SetPriceMethod
+  setGlobalPrice: Dispatch<number | undefined>
+  globalPrice?: number
+  selectedMarkets: ListingMarket[]
+}
+
+const NFTListRow = ({ asset, globalPriceMethod, globalPrice, setGlobalPrice, selectedMarkets }: NFTListRowProps) => {
+  const [expandMarketplaceRows, setExpandMarketplaceRows] = useState(false)
```

Comment:
do you need this separate piece of state. could you just have `expandMarketplaceRows = marketplaces.length >= 2`?

Reply:
a listing can be for multiple marketplaces, so this is to toggle whether a user is splitting the row to view on a per marketplace basis or for generally across all